Gems/Multiplayer/Code/Include/Multiplayer/Components/NetBindComponent.h

```diff
@@ -76,7 +76,17 @@ namespace Multiplayer
         //! IsNetEntityRoleClient
         //! @return true if this network entity is a simulated proxy on a client; otherwise false.
         bool IsNetEntityRoleClient() const;
-        
+
+        //! Sets whether or not a netbound entity is allowed to migrate between hosts.
+        //! Use this feature carefully, as replication is spatially based. If migration is disabled chances are you want to mark the entity as always persistent as well.
+        //! See INetworkEntityManager::MarkAlwaysRelevantToClients and INetworkEntityManager::MarkAlwaysRelevantToServers
+        //! @param value whether to enable or disable host migrations for this entity
+        void SetAllowEntityMigration(EntityMigration value);
+
+        //! Retrieves whether or not the netbound entity is allowed to migrate between hosts.
+        //! @return EntityMigration::Enabled if the entity is allowed to migrate, EntityMigration::Disabled otherwise
+        EntityMigration GetAllowEntityMigration() const;
```

Comment:
Is this an intended part of this change? Migration doesn't seem to be explicitly mentioned in the PR.

Reply:
Ah yes, it's important.. I'll add it to the PR notes, thanks!